library/tiqr/Tiqr/Service.php

```diff
@@ -321,14 +323,31 @@ public function generateAuthQR(string $sessionKey): void
     }
 
     /**
-     * Generate a QR image and send it directly to
-     * the browser.
+     * Generate a QR image in PNG format and send it directly to
+     * the PHP output buffer
+     *
+     * You are responsible for sending the "Content-type: image/png" HTTP header when sending this output to a
+     * webbrowser, e.g.: header('Content-type: image/png')
      *
      * @param String $s The string to be encoded in the QR image
      */
     public function generateQR(string $s): void
     {
-        QRcode::png($s, false, 4, 5);
+        try {
+            $options = new \chillerlan\QRCode\QROptions;
```

Comment:
Maybe 'import' the library classes to prevent having to state them by their FQN?